docs/how-to/freeze-checkbox-version.rst

```diff
@@ -0,0 +1,218 @@
+
+Once snapshots are enabled for a repository, it is possible to pass a specific
+Snapshot ID to most apt or apt-get commands with ``--snapshot [Snapshot ID]`` or
+``-S [Snapshot ID]``, for example:
+
+.. code-block:: bash
+
+   $ apt update --snapshot 20240416T000000Z
+   $ apt policy checkbox-ng -S 20240416T000000Z
+   $ apt install checkbox-ng --snapshot 20240416T000000Z
+
+
+Using a specific Snapshot ID for all apt commands
+`````````````````````````````````````````````````
+
+It is possible to set apt to use a particular snapshot for all apt
+commands of a PPA repository. To do this, the specific Snapshot ID (e.g.
+20240416T000000Z) can be used in place of “yes” in the relevant source.
+
+For Ubuntu 24.04
+''''''''''''''''
+
+.. code-block::
+   
+   # In “/etc/apt/sources.list.d/checkbox-dev-ubuntu-beta-noble.list”
+   Types: deb
+   URIs: https://ppa.launchpadcontent.net/~checkbox-dev/beta/ubuntu/
+   Suites: noble
+   Components: main
+   Signed-By: {KEY}
+   Snapshot: 20240416T000000Z
+
+For Ubuntu 23.10 and earlier
+''''''''''''''''''''''''''''
+
+.. code-block::
+
+   # In “/etc/apt/sources.list.d/checkbox-dev-ubuntu-beta-jammy.list”
+   deb [snapshot=20240416T000000Z] https://ppa.launchpadcontent.net/~checkbox-dev/beta/ubuntu jammy main
+
+
+Disable Snapshot Service for a repository
+`````````````````````````````````````````
+
+For Ubuntu 24.04
+''''''''''''''''
+
+For Ubuntu 24.04 and later, snapshots are enabled automatically for supported
+repositories. If you want to disable them for the Checkbox repository, edit the
+sources file To include ``Snapshot: no``.
+
+.. code-block::
+
+   # In “/etc/apt/sources.list.d/checkbox-dev-ubuntu-beta-noble.list”
+   Types: deb
+   URIs: https://ppa.launchpadcontent.net/~checkbox-dev/beta/ubuntu/
+   Suites: noble
+   Components: main
+   Signed-By: {KEY}
+   Snapshot: no
+
+
+For Ubuntu 23.10 and earlier
+''''''''''''''''''''''''''''
+
+On Ubuntu 23.10 and earlier the included version of apt did not automatically
+detect snapshot support, so snapshots should not be enabled unless you have
+added ``[snapshot=yes]`` to the relevant source.
+
+Using snapshots for 18.04 or earlier
+------------------------------------
+
+The Ubuntu snapshot service is available for 18.04 (Bionic) and 16.04 (Xenial)
+but the apt version included does not support the ``--snapshot`` option. In this
+case, it is required to set up manually the URL in your sources to point to a
+specific snapshot. This option is also possible for later versions of Ubuntu.
+
+Set manually the URL to the snapshot
+------------------------------------
+
+To set the URL to point to a specific snapshot, you have to edit:
+``/etc/apt/sources.list.d/checkbox-dev-ubuntu-{ppa-name}-{ubuntu-version}.list``
+and change the URL:
+
+* ``ppa.launchpadcontent.net``  ->  ``snapshot.ppa.launchpadcontent.net`` 
+* Append the timestamp to the end of the URL
+
+For example: 
+
+.. code-block:: bash
```

Comment:
```suggestion
.. code-block::
```

Reply:
I am wondering why we don't use `code-block:: console` here?